xero-app-store.yml

```diff
@@ -0,0 +1,260 @@
+  license:
+    name: MIT
+    url: "https://github.com/XeroAPI/Xero-OpenAPI/blob/master/LICENSE"
+servers:
+  - description: Xero App Store API
+    url: "https://api.xero.com/appstore/2.0/"
+paths:
+  "/subscriptions/{subscriptionId}":
+    get:
+      tags:
+        - AppStore
+      operationId: getSubscription
+      summary: Retrieves a subscription for a given subscriptionId
+      parameters:
+        - $ref: "#/components/parameters/subscriptionId"
+      responses:
+        "200":
+          description: Success - return response of unique Subscription object
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Subscription"
+              example: '
+                        {
+                          "currentPeriodEnd": "2021-07-20T03:13:48Z",
+                          "endDate": "2021-07-20T03:13:48Z",
+                          "id": "01b5a6f4-8936-4bfa-b703-830702312b87",
+                          "isTestMode": true,
+                          "organisationId": "fdc5be44-9b3e-4ebb-a0e9-11b9737f9a28",
+                          "plans": [
+                            {
+                              "id": "f617dd59-462f-46a1-9519-1765fd38b160",
+                              "name": "Small",
+                              "status": "ACTIVE",
+                              "subscriptionItems": [
+                                {
+                                  "endDate": "2021-07-20T03:13:48Z",
+                                  "id": "c7336bf6-8a47-4f13-9fc0-82420e6922c8",
+                                  "price": {
+                                    "amount": 50,
+                                    "currency": "AUD",
+                                    "id": "31acefbe-bdb7-4329-84d6-51e9afd95327"
+                                  },
+                                  "product": {
+                                    "id": "56d66073-ff78-497b-a726-ca9d56fdafa3",
+                                    "name": "Small",
+                                    "type": "FIXED"
+                                  },
+                                  "startDate": "2021-07-20T03:13:48Z"
+                                }
+                              ]
+                            }
+                          ],
+                          "startDate": "2021-07-20T03:13:48Z",
+                          "status": "ACTIVE"
+                        }
+                      '
+        "404":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+          description: "When a failure occurs in the endpoint"
+      security:
+        - OAuth2: [marketplace.billing]
+components:
+  securitySchemes:
+    OAuth2:
+      type: oauth2
+      description: For more information
+      flows:
+        authorizationCode:
+          authorizationUrl: "https://login.xero.com/identity/connect/authorize"
```

Comment:
I thought app store billing used client credentials grant - do we need the auth URL here?

Reply:
yes you are correct - I updated this to match actuality. This isn't technically used in the SDK Codegen but good to represent it accurately.